### PR TITLE
Require column names as identifiers for SortByColumns()

### PR DESCRIPTION
**Pull request by VamsiModem**
Remove use of string literals for column names arguments in functions: SortByColumns()

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by bryancroteau-MSFT — APPROVED** [review]

File: src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseHelper.cs [hunk 1]
```diff
@@ -421,6 +421,17 @@ internal static DType ClosestParentScopeTypeForSuggestions(CallNode callNode, In
             }
         }
 
+        internal static IEnumerable<KeyValuePair<string, DType>> GetColumnNameIdentifierSuggestions(DType scopeType)
```
Comment (MikeStall) [inline src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseHelper.cs:424]: >GetColumnNameIdentifierSuggestions [](http://example.com/codeflow?start=65&length=34)

where is this used?

Comment (VamsiModem) [reply src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseHelper.cs:424]: This will be used in PAClient intellisense

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by MikeStall — APPROVED** [review]

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**